config/build/create-babel-presets.js

```diff
@@ -22,7 +22,12 @@ module.exports = function createBabelPresets({platform, format}) {
     }
 
     return [
-        "@babel/preset-typescript",
+        [
+            "@babel/preset-typescript",
+            {
+                allowDeclareFields: true,
```

Comment:
I had to add this to support this syntax in the React classes, it gives us much better types for context in React classes:

```
    static contextType = MathInputI18nContext;
    declare context: React.ContextType<typeof MathInputI18nContext>;
```